Swift/SwiftClosures.playground/Pages/In function.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,35 @@
+//: [Simple closure](@previous)
+
+func repeatFunc(num: Int, f: () -> ()) {
```

Comment:
It would be much better to provide more meaningful external names and perhaps swap parameters. E. g. `func repeatFunc(_ function: () -> (), times: Int)`. Maybe provide different external and internal names, It's up to you. 

Swapping isn't required, 'cause it will make impossible to use this function with a trailing closure. But you should update function's signature to make it more meaningful.